src/main/java/org/fundacionjala/at15/pokemon/Pokemon.java

```diff
@@ -16,19 +17,24 @@ public Pokemon(int newMaxHP, String pokemonName) {
         hitPoints = new HitPoints(newMaxHP);
         this.pokemonName = pokemonName;
         Move tackle = new Tackle();
-        this.movementList[0] = tackle;
+        this.movementList.add(tackle);
     }
 
-    public Move[] getMovementList() {
+    public ArrayList<Move> getMovementList() {
         return movementList;
     }
 
-    public Move getMove(int listPosition) {
-        return movementList[listPosition - 1];
+    //esta clase tiene conflicto
+    public Move getMove(int move) {
+        //int moveDefault = 0;
```

Comment:
this one

Reply:
good observation Saul, I forgot about the comment